tools/hooks/Install.bat

```diff
@@ -1,2 +0,0 @@
-@call "%~dp0\..\bootstrap\python" -m hooks.install %*
```

Comment:
Why was this changed?

Reply:
I didn't touch it, must've gone wrong with a master merge somewhere.